packages/iov-crypto/src/index.ts

```diff
@@ -1,9 +1,11 @@
 export * from "./bip39";
 export { EnglishMnemonic } from "./englishmnemonic";
+export * from "./hash";
```

Comment:
Can we make all new export types explicit?

Reply:
I made them all explicit since I was there anyway.